CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 ## [Unreleased][]
 
-[Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.25.0...HEAD
+[Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.25.1...HEAD
+
+## [0.25.1][] - 2021-02-05
```

Comment:
FYI, we don't set versions in PRs. I'll release as is so that'(s okay but in the future, don't set one please.